site/specs/numbers.json

```
@@ -12489,55 +12535,55 @@
             "content": {
               "application/xml": {
                 "examples": {
-                  "example": {
-                    "description": "example",
-                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<SiteResponse>\n    <Site>\n        <Id>3013</Id>\n        <Name>Raleigh</Name>\n        <Description>Test Gateway</Description>\n        <CustomerName>BW</CustomerName>\n        <Address>\n            <HouseNumber>1600</HouseNumber>\n            <StreetName>PENNSYLVANIA</StreetName>\n            <StreetSuffix>AVE</StreetSuffix>\n            <PostDirectional>NW</PostDirectional>\n            <City>WASHINGTON</City>\n            <StateCode>DC</StateCode>\n            <Zip>20006</Zip>\n            <Country>US</Country>\n            <AddressType>Service</AddressType>\n        </Address>\n        <!-- Following appears only for sites on UC Trunking accounts. -->\n        <UcTrunkingConfiguration>\n            <Type>Seats</Type>\n            <UsageCategory>UC500</UsageCategory>\n        </UcTrunkingConfiguration>\n    </Site>\n</SiteResponse>\n"
+                  "For a Non UC account": {
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<SiteResponse>\n   <Site>\n      <Id>3013</Id>\n      <Name>Raleigh</Name>\n      <Description>Test Gateway</Description>\n      <CustomerName>BW</CustomerName>\n      <Address>\n         <HouseNumber>1600</HouseNumber>\n         <StreetName>PENNSYLVANIA</StreetName>\n         <StreetSuffix>AVE</StreetSuffix>\n         <PostDirectional>NW</PostDirectional>\n         <City>WASHINGTON</City>\n         <StateCode>DC</StateCode>\n         <Zip>20006</Zip>\n         <Country>US</Country>\n         <AddressType>Service</AddressType>\n      </Address>\n   </Site>\n</SiteResponse>"
+                  },
+                  "For a UC account": {
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<SiteResponse>\n   <Site>\n      <Id>3013</Id>\n      <Name>Raleigh</Name>\n      <Description>Test Gateway</Description>\n      <CustomerName>BW</CustomerName>\n      <Address>\n         <HouseNumber>1600</HouseNumber>\n         <StreetName>PENNSYLVANIA</StreetName>\n         <StreetSuffix>AVE</StreetSuffix>\n         <PostDirectional>NW</PostDirectional>\n         <City>WASHINGTON</City>\n         <StateCode>DC</StateCode>\n         <Zip>20006</Zip>\n         <Country>US</Country>\n         <AddressType>Service</AddressType>\n      </Address>\n      <UcTrunkingConfiguration>\n         <Type>Seats</Type>\n         <UsageCategory>UC500</UsageCategory>\n      </UcTrunkingConfiguration>\n   </Site>\n</SiteResponse>"
                   }
                 }
               }
             }
           },
           "404": {
-            "description": "<p>the site id entered does not exist</p>\n",
+            "description": "The site was not found for entered id",
```

```suggestion
            "description": "The site was not found for supplied id",
```

`entered` sounds like you're putting it into a field in a UI